packages/nomad/clickhouse.hcl

```diff
@@ -5,14 +5,25 @@ job "clickhouse" {
 
 
   group "clickhouse" {
+
+    update {
+      max_parallel     = 2
+      min_healthy_time = "30s"
+      healthy_deadline = "4m"
+
+      auto_revert = true
+    }
+
     count = 1
 
     network {
       port "clickhouse" {
         to = 9000
+        static = 9000
```

Comment:
need this for stable networking configuration

Reply:
can you elaborate what do you mean by stable? were some packets dropping without this, i don't understand